src/vuestic-theme/vuestic-components/va-rating/VaRating.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div
+    class="va-rating"
+    :class="{'va-rating--disabled': disabled, 'va-rating--readonly':readonly}"
+  >
+    <div
+      v-if="numbers"
+      class="va-rating__numbers"
+    >
+      <div
+        v-for="item in max"
+        :key="item"
+        :class="{'va-rating__numbers__number--empty' : !compareWithValue(item)}"
+        class="va-rating__numbers__number"
+        :style="{
+        'background-color' : color,
+        'height':getIconSize(),
+        'font-size': getNumbersFontSize(),
+        'width': getIconSize()
+        }"
+        @click="setCurrentValue(item, 1)"
+      >
+        {{item}}
+      </div>
+    </div>
+    <div
+      v-else
+      class="va-rating__icons"
+      @mouseover="isHover = true"
+      @mouseout="isHover = false"
+    >
+      <va-rating-item
+        v-for="item in max"
+        :key="item"
+        :icon="icon"
+        :emptyIcon="getIconClasses()"
+        :value="getStarValue(item)"
+        :iconClasses="getIconClasses(item)"
+        :halfIcon="halfIcon"
+        class="va-rating__icons__icon"
+        :style="{'color':color, 'font-size' : getIconSize()}"
+        @click="setCurrentValue(item, $event)"
+        :size="getIconSize()"
+        @hover="onHover(item, $event)"
+        :hover="item <= hoverItemNumber && isHover"
+        :isHalf="item - value === 0.5"
+        :isRatingHover="isHover"
+        @mouseout.native="onHover(value)"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import VaRatingItem from './VaRatingItem'
+
+export default {
+  name: 'va-rating',
+  components: { VaRatingItem },
+  props: {
+    icon: {
+      type: String,
+      default: 'fa fa-star'
+    },
+    halfIcon: {
+      type: String
+    },
+    emptyIcon: {
+      type: String
+    },
+    value: {
+      type: Number,
+      default: 1
+    },
+    readonly: {
+      type: Boolean
+    },
+    disabled: {
+      type: Boolean
+    },
+    numbers: {
+      type: Boolean
+    },
+    max: {
+      type: Number,
+      default: 5
+    },
+    size: {
+      type: String,
+      default: 'medium'
+    },
+    color: {
+      type: String,
+      default: '#4ae387'
+    }
+  },
+  data () {
+    return {
+      hoverItemNumber: this.value,
```

Comment:
`lastHoverItemNumber` perhaps?

Nothing is hovered when component is just created.